catalyst/registry/__init__.py

```diff
@@ -0,0 +1,243 @@
+"""
```

Comment:
Is it possible to rename `catalyst/registry/registry.py` file to something like `catalyst/registry/registry_factory.py`
and move all logic related to `TRANSFORM`, `Callback`, etc. registries from `.__init__.py` to `catalyst/registry/registry.py`?